src/Core/Content/Product/SearchKeyword/ProductSearchTermInterpreter.php

```diff
@@ -28,13 +29,18 @@ public function __construct(
         private readonly TokenizerInterface $tokenizer,
         private readonly LoggerInterface $logger,
         private readonly AbstractTokenFilter $tokenFilter,
-        private readonly KeywordLoader $keywordLoader
+        private readonly KeywordLoader $keywordLoader,
+        private readonly SearchConfigLoader $configLoader,
     ) {
     }
 
     public function interpret(string $word, Context $context): SearchPattern
     {
-        $tokens = $this->tokenizer->tokenize($word);
+        $config = $this->configLoader->load($context);
+        $minSearchLength = $config[0]['min_search_length'] ?? null;
+
+        /** @phpstan-ignore arguments.count (This ignore should be removed when the deprecated method signature is updated) */
```

Comment:
maybe add @deprecated tag v6.8.0 so we can remember it when remove 6.8.0 feature flag